tailwind.config.js

```diff
@@ -154,6 +154,9 @@ module.exports = {
       tight: 1.2,
       loose: 1.5,
     },
+    maxWidth:{
+      'screen-xl': "1440px", 
+    },
```

Comment:
needed ? we shud not restrict...its already covered in container class

Reply:
container has x padding, this will be of use for any full screen section to allign properly with container if it requires elements outside conainer